packages/react-components/src/util/getFormatedBalance.ts

```diff
@@ -0,0 +1,11 @@
+import { formatBalance } from '@polkadot/util';
+
+
```

Comment:
Add comment.
Some spec tests would be good, it's not really clear how `value` will be transformed 
I think it's truncating the value so input `0.1234` is becoming `0.1230`